src/Components/CreateImageWizard/CreateImageWizard.tsx

```diff
@@ -151,10 +154,12 @@ const CreateImageWizard = ({ isEdit }: CreateImageWizardProps) => {
 
   // =========================TO REMOVE=======================
 
-  // Feature flags
```

Comment:
I wonder if we could just have separate useFlag hooks for on-prem and the service and then import conditionally?

Reply:
Maybe something like the example here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/import#importing_different_modules_based_on_environment